src/Api/IdentifiersExtractor.php

```diff
@@ -67,7 +67,8 @@ public function getIdentifiersFromResourceClass(string $resourceClass): array
     public function getIdentifiersFromItem($item): array
     {
         $identifiers = [];
-        $resourceClass = $this->getObjectClass($item);
+        $resourceClass = null !== $this->resourceClassResolver ? $this->resourceClassResolver->getResourceClass($item) : $this->getObjectClass($item);
```

Comment:
```suggestion
        $resourceClass = null === $this->resourceClassResolver ? $this->getObjectClass($item) : $this->resourceClassResolver->getResourceClass($item);
```

Reply:
This feels weird to me. Having the resource class resolver should be the "nominal" case, so it's more logical that it comes first. Someone reading the code doesn't have to go to the "exceptional" branch to see it.

Reply:
negative condition are hard to read (at least for me). It's considered a bad practice by many quality checking tools: https://pmd.sourceforge.io/pmd-5.0.3/rules/java/design.html#ConfusingTernary

Reply:
I disagree that it's "hard to read". That seems entirely subjective. For me it's easier to read in this case, because what it says is: "If we have the resource class resolver, let's do the right thing, otherwise we use a fallback."

Reply:
If you'd like, I could change `null !== $this->resourceClassResolver` to simply `$this->resourceClassResolver`